src/app.js

```diff
@@ -116,7 +119,8 @@ setGTM({
 });
 
 router.use(getGTM);
-router.use(getAssetPath);
+router.use(getLanguageToggle);
+//router.use(getAssetPath);
```

Comment:
How come this is commented out?

Reply:
I think this was accidental. it has been fixed